docs/pages/docs/guides/auth-and-access-control.mdx

```diff
@@ -136,7 +136,7 @@ You configure it like this:
 ```ts
 const { withAuth } = createAuth({
   // ...
-  sessionData: 'name isAdmin',
+  sessionData: 'id isAdmin',
```

Comment:
When using `createAuth`, `session.itemId` is actually the `id` of the authenticated list.
This is redundant.

See https://github.com/keystonejs/keystone/blob/43a73e5bd6fddd0404dad87ab3ff406aef6143e9/packages/auth/src/index.ts#L242-L245

This documentation needs work,  we need to outline these interfaces and their guarantees properly.

Reply:
```suggestion
  sessionData: 'isAdmin',
```